src/utils/hooks/useGetReviewInfo.tsx

```diff
@@ -23,6 +23,7 @@ const useGetReviewInfo = ({ applicationId, userId }: UseGetReviewInfoProps) => {
     notifyOnNetworkStatusChange: true,
     // if this is removed, there might be an infinite loading when looking at a review for the frist time, after clearing cache
     // it's either this or removing 'totalCount' in `reviewQuestionAssignments` from this query
+    // ended up removing totalCount from query and keeping this as nextFetchPolicy (was still seeing glitched with totalCount and had "can't update unmounted component error")
     fetchPolicy: 'network-only',
```

Comment:
Maybe can be removed now?

Reply:
i think it needs to be network only, since it needs to wait for the trigger. plus i had constant issues with cache after database re-initilisation

Reply:
i can remove the comment, but thought it was useful